Sources/BraintreeCore/BTHTTP.swift

```diff
@@ -179,6 +179,7 @@ class BTHTTP: NSObject, NSCopying, URLSessionDelegate {
         method: String,
         path: String,
         parameters: [String: Any]? = [:],
+        headers: [String: String]? = [:],
```

Comment:
It looks like in other places we are defaulting to `nil`. Is there a reason we want to default to an empty dictionary for this method?

Reply:
ahh good catch! yea there's no particular reason - updated it to `nil` instead